tests/test_integration/test_code_generation.py

```diff
@@ -97,3 +98,20 @@ def test_stage_without_effect(backend):
     def definition(field_a: gtscript.Field[np.float_]):
         with computation(PARALLEL), interval(...):
             field_c = 0.0
+
+
+@pytest.mark.parametrize(
+    ["name", "backend"], itertools.product(stencil_definitions.names, CPU_BACKENDS)
+)
+def test_cache_info(name, backend):
+    stencil_definition = stencil_definitions[name]
+    externals = externals_registry[name]
+    stencil = gtscript.stencil(backend, stencil_definition, externals=externals)
+
+    module = stencil.options["module"]
+    qualified_name = f"{module}.{name}"
+    version = (str(stencil.__class__).split("_")[-1])[0:-2]
```

Comment:
The stencil_id version is available in the generated StencilObject as `stencil_class._gt_id_`. It should be read from there instead of hacking the class name.

Reply:
Thank you, I was not ware of `stencil_class._gt_id`.